lms/static/scripts/frontend_apps/components/LMSFilePicker.js

```diff
@@ -118,11 +161,7 @@ export default function LMSFilePicker({
   // On the initial load, fetch files or prompt to authorize if we know that
   // authorization will be required.
   useEffect(() => {
-    if (dialogState.state === 'authorizing') {
-      authorizeAndFetchFiles();
-    } else {
-      fetchFiles();
-    }
+    fetchFiles();
```

Comment:
Is this change still applicable given that we're reverting the authorization error-handling changes in this PR?

Reply:
This useEffect hook is only run when the component is mounted. At that time, the `dialogState.state` is never 'authorizing'.

This is my understanding on these lines. That's why I remove the lines from 121 to 125.

Reply:
This makes sense. Yesterday I missed the fact that `dialogState.state` is always `fetching` here.